main.tf

```diff
@@ -146,7 +146,7 @@ resource "aws_cloudwatch_metric_alarm" "alarm_cpu_no_lb" {
   count = var.cloudwatch_alarm_cpu_enable && ! (var.associate_alb || var.associate_nlb) ? 1 : 0
```

Comment:
Same here. shouldn't this be alarm_mem_enable ?

Reply:
I think in this case it should be `cloudwatch_alarm_cpu_enable` since it is within the `alarm_cpu_no_lb` resource.

Reply:
you're right!